vortex-scalar/src/serde/flatbuffers.rs

```diff
@@ -17,9 +17,7 @@ impl TryFrom<fb::Scalar<'_>> for Scalar {
         let dtype = value.dtype();
         let dtype = DType::try_from(dtype)?;
 
-        // TODO(ngates): what's the point of all this if I have to copy the data into a Vec?
-        let flex_value = value.value().flex().iter().collect_vec();
-        let reader = flexbuffers::Reader::get_root(flex_value.as_slice())?;
+        let reader = flexbuffers::Reader::get_root(value.value().flex().bytes())?;
```

Comment:
I figured out that `flatbuffers::Vector::bytes` is the contiguous view of bytes that back the vector. So it can be transmuted into whatever primitive T there is, which in this case is already u8.